src/scalper_utils.ts

```diff
@@ -286,7 +286,7 @@ export async function getFairValue(
   symbol: string,
   owner: Keypair
   ) {
-  let fairValue: number;
+  let fairValue = 0;
```

Comment:
is 0 just the value for no oracle price?
should document that or switch to undefined

Reply:
Correct, we pass fairvalue 0 if none of the oracles return a price. This was solving another break that can't reproduce now that the oracles are behaving